rpc/eth_api.go

```diff
@@ -29,6 +28,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/eth"
```

Comment:
so it seems that using this library is causing some issues when trying to build Ethermint. Can you take a look @araskachoi 🙏 ?

Reply:
hmm.. weird how it breaks when building in docker but works when you build binaries manually. Will keep digging tho

Reply:
@fedekunze can we try to merge the refactored code that we merged and see if that fixes this?